src/components/PupilComponents/PupilUnitsSection/PupilUnitsSection.tsx

```diff
@@ -98,20 +101,44 @@ export const PupilUnitsSection = ({
       phase={phase}
       titleSlot={titleSlot}
       filterSlot={filterSlot}
-      counterSlot={
+      subheadingSlot={
```

Comment:
Didn't use <OakPupilJourneyListItemSubheading /> or <OakPupilJourneyListCounter /> here as the OakPupilJourneyListItemSubheading showed bullets even when the listItems array was empty, which meant that it needed another update in OakComponents in order to work. In favour of Simon's previous PR comment regarding over composition I've opted to use our basic components here instead.

Reply:
Makes sense. Brings me back to whether we need a tidy up on OakComponents